tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -41,3 +54,26 @@ Given('the quota of user {string} has been set to {string}', function (userId, q
   )
     .then(res => httpHelper.checkStatus(res, 'Could not set quota.'))
 })
+
+Given('these users have been created with default attributes but not initialized:', function (dataTable) {
```

Comment:
Currently, we have the same method for `user has been created with default attributes`, `user has be created but not initialized`, `user has been created`.
Later we have to implement proper methods for them.